binaries/llm-cli/src/cli_args.rs

```diff
@@ -332,6 +332,14 @@ pub struct ModelLoad {
     #[arg(long, short = 'm')]
     pub model_path: PathBuf,
 
+    /// Where to load the vocabulary from
+    #[arg(long, short = 'v')]
+    pub vocabulary_path: Option<PathBuf>,
+
+    /// Where to load the vocabulary from
```

Comment:
```suggestion
    /// Remote vocabulary URL
```

Reply:
Actually in retrospective this should probably specify that it's expected to be a HuggingFace URL